src/rules/no-important.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * @fileoverview Rule to prevent !important in CSS.
+ * @author Yann Bertrand
+ */
+
+//-----------------------------------------------------------------------------
+// Rule Definition
+//-----------------------------------------------------------------------------
+
+export default {
+	meta: {
+		type: /** @type {const} */ ("problem"),
+
+		docs: {
+			description: "Disallow important annotations",
+			recommended: true,
+		},
+
+		messages: {
+			forbiddenImportantUsage: "!important annotations are not allowed.",
+		},
+	},
+
+	create(context) {
+		return {
+			"Declaration[important=true]"(node) {
+				context.report({
+					loc: {
+						start: node.loc.start,
+						end: {
+							line: node.loc.start.line,
+							column:
```

Comment:
This will just highlight the property name. I think it would be better to find the location of the `!important` and set that as the location of the error.